projects/canopy/src/styles/styles.scss

```diff
@@ -7,3 +7,62 @@
 @import './typography';
 @import './utils';
 @import './variants';
+
+.lg-skeleton {
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  flex-flow: column;
+  background-position: center;
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-image: linear-gradient(
+    var(--color-platinum) calc(100% - 0.063em),
+    transparent 0
+  );
+  text-indent: -9999em;
+  max-width: 100%;
+}
+
+.lg-skeleton--right {
+  align-self: flex-end;
+  text-indent: 9999em;
+}
+
+.lg-skeleton:before {
+  content: '\200B';
```

Comment:
This is the `&nbsp;` I mentioned that gives the element dimensions so that the consumer wouldn't have to manually specify height/width.